package.json

```diff
@@ -25,15 +27,19 @@
     "jquery": "3.3.1",
     "jquery-validation": "1.16.0",
     "katex": "0.8.3",
+    "mini-css-extract-plugin": "0.4.0",
     "moment": "2.21.0",
     "moment-timezone": "0.5.14",
+    "node-sass": "4.8.3",
     "perfect-scrollbar": "1.3.0",
     "plotly.js": "1.19.2",
+    "sass-loader": "7.0.1",
     "script-loader": "0.7.2",
     "source-map-loader": "0.2.3",
     "string.prototype.codepointat": "0.2.0",
     "string.prototype.endswith": "0.2.0",
     "string.prototype.startswith": "0.2.0",
+    "style-loader": "0.21.0",
```

Comment:
I think you could do the `webpack` and `yarn.lock` changes in a separate first commit, so that we can merge those (since they shouldn't break anything) while we iterate on the rest of the configuration.

Reply:
Moved to separate commit.